webpack.dev.js

```diff
@@ -0,0 +1,15 @@
+const merge = require('webpack-merge');
+const config = require('./webpack.config.js');
+const WriteFilePlugin = require('write-file-webpack-plugin');
+
+module.exports = merge(config, {
+  mode: 'development',
+  devtool: 'source-map',
+  plugins: [
+    ...config.plugins,
+    new WriteFilePlugin()
```

Comment:
You shouldn't need a plugin to write your bundle to disk when running `webpack -w` without the dev server. In this case it will have no effect

Reply:
We are using `webpack-dev-server` instead of `webpack` for the start script. See: https://github.com/standardnotes/web/pull/398#discussion_r420507179

Reply:
I understood that for the web repo but for snjs it seems we're only using vanilla webpack: https://github.com/standardnotes/snjs/blob/354dee8a881318492bf466370a6f200564577c89/package.json#L7